application/pm.py

```diff
@@ -246,14 +246,18 @@ def load_feature_tracking(main_window):
     """
     Loads feature_tracking information into the specified main window.
     """
-    main_window.feature_tracking_video_player.loadVideo(ac.CURRENT_PROJECT_VIDEO_PATH)
+    #main_window.feature_tracking_video_player.loadFrames(ac.CURRENT_PROJECT_VIDEO_PATH)
```

Comment:
Probably want to completely remove this commented code? Version control means we don't need to keep it?